src/app/helptext/system/ssh-keypairs.ts

```diff
@@ -0,0 +1,14 @@
+import { T } from '../../translate-marker';
+
+export default {
+    name_placeholder: T('Name'),
+    name_tooltip: T(''),
+
+    private_key_placeholder: T('Private Key'),
+    private_key_tooltip: T(''),
```

Comment:
Will the private key be visible after generation?
```suggestion
    private_key_tooltip: T('See <i>Public key authentication</i> in <a href="https://www.freebsd.org/cgi/man.cgi?query=ssh" target="_blank">SSH/Authentication</a>.'),
```

Reply:
@wblock yes. the private key is visible.